mpf/core/utility_functions.py

```diff
@@ -335,11 +335,18 @@ def event_config_to_dict(config) -> dict:
         return return_dict
 
     @staticmethod
-    def int_to_hex_string(source_int: int) -> str:
-        """Convert an int from 0-255 to a one-byte (2 chars) hex string, with uppercase characters."""
+    def int_to_hex_string(source_int: int, allow_overflow: bool = False) -> str:
+        """Convert an int from 0-255 to a one-byte (2 chars) hex string, with uppercase characters.
```

Comment:
Maybe we should have two functions? This one seems to be named imprecisely. The doc block still states that it only converts to 2 chars. Maybe "int_to_hex_string" for a generic one. And "one_byte_to_hex_string" (or similar)?

Reply:
Agreed. The function is limited to just FAST platform devices, but there are a lot of instances of its use. I'm happy to file this for refactoring in a separate PR, to keep the changes clean.